docs/APIBaselines.md

```diff
@@ -61,6 +50,21 @@ Two new entry needs to be added to the PublicAPI.Unshipped.txt file for an updat
 Microsoft.AspNetCore.DataProtection.Infrastructure.IApplicationDiscriminator.Discriminator.get -> string?
 ```
 
+### Steps for adding and updating APIs
+
+1. Update AspNetCore.sln and relevant `*.slnf` file to include the new project
+1. `{directory containing relevant *.slnf}\startvs.cmd`
+1. F6 *or whatever your favourite build gesture is*
+1. Click on a RS0016 (or whatever) error
+1. Right click in editor on underscored symbol or go straight to the “quick fix” icon to its left. Control-. also works.
+1. Choose “Add Blah to public API” / “Fix all occurrences in … Solution”
+1. Click Apply
+1. F6 *again to see if the fixer missed anything or if other RS00xx errors show up (not uncommon)*
+1. Suppress or fix other problems as needed but please suppress (if suppressing) using attributes and not globally or with `#pragma`s because attributes make the justification obvious e.g. for common errors that can't be fixed
+    `[SuppressMessage("ApiDesign", "RS0026:Do not add multiple public overloads with optional parameters", Justification = "Required to maintain compatibility")]`
+    or
+    `[SuppressMessage("ApiDesign", "RS0027:Public API with optional parameter(s) should have the most parameters amongst its public overloads.", Justification = "Required to maintain compatibility")]`
+
```

Comment:
Below the steps or as part of step (1), mention skipping the first step when updating API baselines

Reply:
I think it's pretty obvious that the step can be skipped if already done.